util/test/test_query.py

```diff
@@ -735,6 +736,32 @@ def test_populateNftAssetNames():
     assert nfts[0].name == "Take a Ballet Lesson"
 
 
+@enforce_types
+def _test_queryPassiveRewards(addresses):
+    chain = brownie.network.chain
+    S_PER_WEEK = 604800
+    fee_distributor = oceanutil.FeeDistributor()
+    OCEAN = oceanutil.OCEANtoken()
+
+    OCEAN.transfer(
+        fee_distributor.address,
+        toBase18(1000.0),
+        {"from": brownie.accounts[0]},
+    )
+    fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
+    fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
+    chain.sleep(S_PER_WEEK * 3)
+    chain.mine()
+    fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
+    fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
+    balances, rewards = query.queryPassiveRewards(CHAINID, chain.time(), addresses)
+    print(balances)
+    print(rewards)
+    assert balances[0] == balances[1]
+    assert rewards[0] == rewards[1]
+    assert rewards[0] > 0
+
```

Comment:
I'd suggest that we should also have a test that reads the rewards and balances out of the local csv, so that we can be sure the the csv has successfully been saved in the right location with the right name and headings.